### ISSUE TITLE
Do not tell users to disable mixed content

### ISSUE THREAD

**Issue by DamonGant** [EchoDev/TwitchHTML5#37]
This is bad.

Seriously bad.

There needs to be a different solution here. A plugin that whitelists twitch to do that for instance. You can't just provide instructions for people that don't know better that bascially disable any protection TLS offers. One intercepted JS file can control an entire page shown as secure.

**Comment by EchoDev** [comment]
HTTPS content still goes over HTTPS, allowing mixed content doesn't change that. If there is mixed content on a page the page is not shown as secure, it is shown as insecure. 


Also users are warned about this in the installation guide.

**Comment by onetimesignup** [comment]
If you're refering to Chrome then surely you could just create one shortcut to launch it with mixed content and use it for twitch and one shortcut that does not and use it for the rest of the web.

As for firefox it's beeing more selective when it comes to what pages you're allowing mixed content on, at least if youre using the built-in padlock, chrome used to have this as well but as a shield on the right side of the url-field but for some reason they broke or removed this around v46-47 and I haven't seen them properly restore it yet (i', on v50 where they promised it would be working but where it's not, I haven't updated since either btw so maybe it is now maybe it isn't)

**Comment by EchoDev** [comment]
Can Chrome start with specific options per window/process? I thought it used the start up options globally for all windows/processes.

With Firefox I'm 100% sure you can manually enable mixed content per page (Which for some users might be too annoying)

**Comment by onetimesignup** [comment]
You're correct, I just assumed seeing as how chrome launches it's separate process/instances along with having while set of different rules for incognito etc etc that running multiple -different- instances of chrome wouldn't be a problem.

I experimented a bit with this,
and for some reason they prefer making it a problem.

This is why I loved opera presto and running different versions of it, I could easily setup different browser enviroments that wasnt trying t o fight or win over the other instance of the browser, shit just worked.

but again even if you got this working; this shouldnt be needed.

All you had to do in the past was to click the shield that is now longer visible and it would give you the option to allow mixed content for the site that you were currently viewing.

**Comment by EchoDev** [comment]
So baseline is:

I could change the description for Firefox for manually allowing the mixed content but Chrome doesn't allow whitelisting of a single site/page if I'm reading correctly?

**Comment by onetimesignup** [comment]
if it was my call then i would say that the plan a for chrome users is to downgrade or stay on v47 until google re-enables mixed content permission options for iframes, if they cant have that then the --allow-mixed-content is plan b

Ive been looking for a whitelist myself but all i found or heard was a blacklist and im not even sure if that even applies to this problem or if it only was about https certs.

I also read something about configuring chrome around different profiles and different versions but even there i wasnt sure if it was going to work and im honestly too tired to mess with it.

so plab A is the route i'll be going with on my setup of google chrome.

**Comment by onetimesignup** [comment]
that said, stupid filehippo redirects users to download the online installer for latest version of chrome.

nobody wins anything
